src/app/Pages/Scan/scan.module.css

```diff
@@ -0,0 +1,57 @@
+.container {
+  display: grid;
+  place-items: center;
+  gap: 1rem;
+  height: 100vh;
+  align-content: center;
+}
+
+.logo {
+  width: 3rem;
+}
+
+.headline {
+  color: var(--color-primary);
+  font-weight: var(--font-weight-headline);
+  font-size: 3rem;
+  padding-bottom: 6rem;
+}
+
+.subtitle {
+  color: var(--color-primary);
+}
+
+.scannedText {
+  border: var(--border-primary);
+  padding: 3rem;
+  border-radius: var(--border-radius);
+}
+
+.loading {
+  overflow: hidden;
+}
+
+progress {
```

Comment:
Be careful using element selectors. This effects every `progress` element in your project.